tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -46,17 +47,20 @@ def get_infra_stack_outputs(stack_name):
     }
 
 
-def get_ad_config_param_vals(stack_name, bucket_name, password_secret_arn):
+def get_ad_config_param_vals(directory_stack_name, nlb_stack_name, bucket_name, password_secret_arn):
     """Return a dict used to set values for config file parameters."""
-    infra_stack_outputs = get_infra_stack_outputs(stack_name)
+    stack_outputs = {
+        stack_type: get_infra_stack_outputs(stack_name)
+        for stack_type, stack_name in [("directory", directory_stack_name), ("nlb", nlb_stack_name)]
```

Comment:
minor: why using a look instead of laying out the dict directly? Or maybe just use two variables `directory_stack_outputs` and `nlb_stack_outputs`?

Reply:
i don't have a great reason. i think the idea was to use one line instead of two, but i agree looking at it now that it would probably be simpler to read if i used two variables instead. changing it now.

Reply:
fixed.